<launch>
    <node pkg="pedestrian_detector" type="loadBackground" name="crossing_background" output="screen">
	    <param name="img_file" value="$(find pedestrian_detector)/crossing_figure3.png"/>
    </node>
    <node pkg="pedestrian_detector" type="visualizer" name="visualizer_dataAssoc" output="screen">
	    <param name="image_transport" value="compressed"/>
	    <param name="ROI_text" value="true"/>
	    <param name="ROI_rect" value="true"/>
	    <param name="threshold" value="0.0"/>
	    <param name="speedchart_bg" value="$(find pedestrian_detector)/blank.png"/>
	    <param name="beliefchart_bg" value="$(find pedestrian_detector)/blank.png"/>
	    <remap from="image_raw" to="/camera_front/image_raw"/>
	    <remap from="veri_pd_vision" to="ped_data_assoc"/>
	</node>
</launch>
